docs/modules/query/pages/querying-maps-sql.adoc

```diff
@@ -78,7 +78,14 @@ The `this` alias is returned by a query only if the value does not contain an ob
 If you store objects in a map, you can query its fields only if the object is serialized using one of the following options:
 
 - For Java objects (xref:serialization:implementing-java-serializable.adoc[`Serializable`], xref:serialization:implementing-dataserializable.adoc[`DataSerializable`], xref:serialization:implementing-dataserializable.adoc#identifieddataserializable[`IdentifiedDataSerializable`]), only public fields and getters are available. A public field is available with its exact name. A getter is available with the JavaBean naming conventions.
-- For xref:serialization:implementing-portable-serialization.adoc[`Portable`] objects, the fields that are written in the `Portable.writePortable()` method are exposed using their exact names.
+- For xref:serialization:implementing-portable-serialization.adoc[`Portable`] objects, 
+	- the fields that are written in the `Portable.writePortable()` method are exposed using their exact names.
+	- the fields that are written in the `GenericRecordBuilder` API are exposed using their exact names. 
+- For xref:serialization:compact-serialization.adoc[`Compact`] objects, 
+	- the fields that are written in the `CompactSerializer.write()` method are exposed using their exact names.
+	- the fields that are written in the `GenericRecordBuilder` API are exposed using their exact names. 
+	- From members or from Java client, one can write `Compact` objects with zero-config. In that case, the fields will be exposed similar to Java Objects described above. Only public fields and getters are available. A public field is available with its exact name. A getter is available with the JavaBean naming conventions.
```

Comment:
This nested list is not properly displayed on https://deploy-preview-151--hardcore-allen-f5257d.netlify.app/hazelcast/5.0-snapshot/query/querying-maps-sql.html#querying-object-fields

Reply:
Fixed it.